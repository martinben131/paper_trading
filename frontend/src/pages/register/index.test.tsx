import React from "react";
import { shallow } from "enzyme";
import Register from "./index";

test("Renders register page correctly", () => {
  const component = shallow(<Register />);
  expect(component).toMatchSnapshot();
});
