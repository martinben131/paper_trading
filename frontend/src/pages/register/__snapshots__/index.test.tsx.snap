// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders register page correctly 1`] = `
<Fragment>
  <h4
    className="bp3-heading"
  >
    Create account
  </h4>
  <Blueprint3.FormGroup>
    <Blueprint3.InputGroup
      id="username"
      onChange={[Function]}
      placeholder="Enter username here"
    />
    <br />
    <Blueprint3.InputGroup
      id="email"
      onChange={[Function]}
      placeholder="Enter email here"
      type="email"
    />
    <br />
    <Blueprint3.InputGroup
      id="password"
      onChange={[Function]}
      placeholder="Enter password here (must be at least 6 characters)"
      type="password"
    />
    <br />
    <Blueprint3.InputGroup
      id="confirm_password"
      onChange={[Function]}
      placeholder="Confirm password"
      type="password"
    />
    <br />
    <Component>
      Choose desired starting balance! (More than $0)
    </Component>
    <Blueprint3.Slider
      disabled={false}
      initialValue={0}
      intent="primary"
      labelRenderer={[Function]}
      labelStepSize={10000}
      max={100000}
      min={0}
      onChange={[Function]}
      showTrackFill={true}
      stepSize={1000}
      value={0}
      vertical={false}
    />
    <br />
    <Blueprint3.Button
      disabled={true}
      intent="success"
      onClick={[Function]}
      text="Submit"
    />
  </Blueprint3.FormGroup>
</Fragment>
`;
